<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridOptions</class>
 <widget class="QWidget" name="GridOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0">
   <item row="6" column="1">
    <widget class="QSpinBox" name="w_spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Grid options</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="grid_checkBox">
     <property name="text">
      <string>Show grid</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="bounds_checkBox">
     <property name="text">
      <string>Show bounds</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="unregister_members_checkBox">
     <property name="text">
      <string>Unregister members</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="auto_load_checkBox">
     <property name="text">
      <string>Auto load</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="h_spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QPushButton" name="auto_resize">
     <property name="text">
      <string>Auto resize</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="auto_register_checkBox">
     <property name="text">
      <string>Auto register</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="height">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="width">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
